app/src/ui/thank-you/thank-you.tsx

```diff
@@ -49,7 +49,7 @@ export class ThankYou extends React.Component<IThankYouProps, {}> {
   private renderConfetti(): JSX.Element | null {
     const confetti = new Array<JSX.Element>()
 
-    const howMuchConfetti = 1500
+    const howMuchConfetti = 750
```

Comment:
Shouldn't this also be changed?

https://github.com/desktop/desktop/blob/bf8c4170a368cc5726ee538224f34010d527151f/app/styles/ui/dialogs/_thank-you.scss#L126

Reply:
Great catch! This wouldn't have hurt anything - but makes sense for them to match.